apps/docs/src/guide/quickstart/index.md

```diff
@@ -0,0 +1,87 @@
+# Quickstart
+
+You can quickly bootstrap a full-stack Fuel project locally with the following command:
+
+::: code-group
+
+```sh [pnpm]
+pnpm create fuels
+```
+
+```sh [bun]
+bun create fuels
+```
+
+```sh [npm]
+npm create fuels
+```
+
+```sh [yarn]
+yarn create fuels
+```
+
+:::
+
+This will setup a new full-stack Fuel project. To get things running, you'll need to install the dependencies and start the development servers:
+
+::: code-group
+
+```sh [pnpm]
+# (Make sure you cd into your new project directory first!)
+pnpm install
+
```

Comment:
When I `cd` to the project dir and run `pnpm install`, I am running into an error.
<img width="861" alt="Screenshot 2023-12-18 at 19 09 06" src="https://github.com/FuelLabs/fuels-ts/assets/30977845/27eb7dc2-c1ef-430a-92c0-8b2bbb027902">

I wonder if I am missing something.

Reply:
Same as the previous comment. Please use the pr release tag and it should work fine